code/GraphQL/_legacy/FieldFormatter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace SilverStripe\Admin\GraphQL;
+
+use SilverStripe\Core\Convert;
+
+class FieldFormatter
```

Comment:
```suggestion
/**
 * @internal Use GraphQL v4 instead
 */
class FieldFormatter
```

Reply:
OK, it seemed a bit overkill to me to mark the classes themselves as internal. What we really care about is _are you using them in the schema_, which is why the setters are internal (the only way to use them).